.github/workflows/release.yml

```diff
@@ -27,6 +27,12 @@ jobs:
       uses: actions/setup-go@v2
       with:
         go-version: '1.20'
+    - name: Install Snapcraft
+      uses: samuelmeuli/action-snapcraft@v1
+    - name: Setup Snapcraft
+      run: |
+        mkdir -p $HOME/.cache/snapcraft/download
+        mkdir -p $HOME/.cache/snapcraft/stage-packages
```

Comment:
This step feels odd. Why doesn't the action above (or the `snap` command create these)? It doesn't seem you're using this cache between runs either?  (we don't need, just asking.)

Reply:
These couple lines are necessary because of [this](https://github.com/goreleaser/goreleaser/issues/1715) problem. If it doesn't exist when snapcraft command is executed things go wild. Kidding. It just crash and burns. :)

Reply:
I had one more link about this as a comment in the file, but I think I lost that commit. :/

Reply:
Yeah, let's add a comment as to why we're adding this step. I definitely won't remember this in a few months :)

Reply:
Done

Reply:
@ahmetb Do you have any other considerations?

Reply:
I clicked on "Run action" and CI seems to be failing. I think we need a way to configure goreleaser to skip snapcraft step while just building/packaging (and skip publishing).

Reply:
I saw it. I already have an idea

Reply:
@ahmetb Done. Please check again.